app/controllers/api/v1/demographics/districts_controller.rb

```diff
@@ -4,13 +4,16 @@ class Api::V1::Demographics::DistrictsController < Api::ApiController
   def show
     district = District.find_by(slug: params["slug"])
     year = SchoolYear.find_by(years: params["year"])
+    key = params[:api_key]
     if invalid_request(district, year)
       message = invalid_request(district, year)
```

Comment:
Totally agree with @amaxwellblair that you could one line that if you wanted. On the other hand, I tend to take the opposite path. I am more verbose and use more white space and more of everything. I just find that trying to dissect a one liner when something is breaking at 1am is just not worth it to me.

For the record, a lot of people disagree with my point of view on this. 😆